hummingbot/connector/exchange/trading212/trading212_constants.py

```
@@ -0,0 +1,74 @@
+from decimal import Decimal
```

Flake8 (7.2.0)

[error] 'decimal.Decimal' imported but unused

(F401)